app/src/androidTest/java/com/android/unio/ui/event/EventDetailsTest.kt

```diff
@@ -0,0 +1,185 @@
+package com.android.unio.ui.event
+
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.isNotDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performScrollTo
+import androidx.navigation.NavHostController
+import com.android.unio.model.association.Association
+import com.android.unio.model.association.AssociationCategory
+import com.android.unio.model.event.Event
+import com.android.unio.model.event.EventListViewModel
+import com.android.unio.model.event.EventRepository
+import com.android.unio.model.firestore.firestoreReferenceListWith
+import com.android.unio.model.user.User
+import com.android.unio.model.user.UserRepository
+import com.android.unio.model.user.UserViewModel
+import com.android.unio.ui.navigation.NavigationAction
+import com.google.firebase.firestore.CollectionReference
+import com.google.firebase.firestore.FirebaseFirestore
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.MockitoAnnotations
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+
+class EventDetailsTest {
+  private lateinit var navHostController: NavHostController
+  private lateinit var navigationAction: NavigationAction
+  @Mock private lateinit var collectionReference: CollectionReference
```

Comment:
It appears that several of these mocked variables such as `db` or `collectionReference` are not used in this test. Consider removing them if they are not necessary.